EC-84: Revise Cache, Catalog and Checkout sections (#3)
ezsystems/commerce-documentation: docs/guide/catalog/catalog_faq.md

# Catalog FAQ

## How can I change the standard image-not-available.png image?

Define the path to the image as a parameter:

``` yaml
parameters:
    silver_tools.default.defaultImage: ../web/bundles/<yourprojectbundle>/img/image-not-available.png
```

Create a new image in your bundle:

`src/<project>/Bundle/<BundleName>/Resources/public/img/image-not-available.png`

Remove the already generated image first:

``` bash
rm web/var/ecommerce/storage/images/thumb_medium/i/m/a/-image-not-available_thumb_medium.png
```

> ## Do the prices set up in the Back Office include VAT?

Reviewer: ```suggestion
## How to include VAT in the prices from the Back Office?
```
